hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java

```diff
@@ -311,15 +319,24 @@ public FSDataOutputStream create(final Path f, final FsPermission permission, fi
 
     statIncrement(CALL_CREATE);
     trailingPeriodCheck(f);
+    TracingContext tracingContext = new TracingContext(clientCorrelationId,
+            fileSystemId, FSOperationType.CREATE, overwrite, tracingHeaderFormat, listener);
 
     Path qualifiedPath = makeQualified(f);
+    boolean fileOverwrite = overwrite;
```

Comment:
Add comments that this is needed for create idempotency when overwrite is false.

Reply:
Done.